app/services/notifications/new_comment/send.rb

```diff
@@ -93,7 +93,16 @@ def send_push_notifications(user_id, title, subtitle, body, path)
               }
             },
             data: {
-              url: "https://dev.to" + path
+              url: App.url(path)
+            }
+          },
+          fcm: {
+            notification: {
+              title: title,
+              body: subtitle
+            },
+            data: {
+              url: App.url(path)
```

Comment:
Why does this get repeated?

Reply:
just an illusion because of the idff. The first one @fdoxyz changed is for iOS notifications, this one is in the Android payload

:magic:

Reply:
Because of codeclimate 25 line limit (it used to look [like this](https://github.com/thepracticaldev/dev.to/pull/6883/commits/18dd1233534127734bcced12cfbac3ac97862d82#diff-6aa967016aaf68d17a712f933a3a134dR86)).

I wasn't sure if it was okay to repeat them, but I agree it does stick out quite a bit and it's an unnecessary second call to the method. Now that you ask though I got an idea on a way to clean it up a bit with fewer lines: Pushing the change in a sec